src/tooling/piral-cli/src/injectors/pilet.ts

```diff
@@ -20,20 +20,22 @@ export interface PiletInjectorConfig extends KrasInjectorConfig {
 export default class PiletInjector implements KrasInjector {
   public config: PiletInjectorConfig;
   private port: number;
+  private https: boolean;
 
   constructor(options: PiletInjectorConfig, config: KrasConfiguration, core: EventEmitter) {
     this.config = options;
     this.port = config.port;
+    this.https = !!config.ssl;
```

Comment:
Yes this one adds it already, so no need to have it in the injector config, too.

Reply:
Thanks for pointing right